src/lib/banner/banner.ts

```diff
@@ -35,10 +35,12 @@ declare global {
  *
  * @summary Banners are used to inform users of important information, such as errors, warnings, or success messages.
  *
+ * @property {boolean} [canDismiss=false] - Determines if a banner can be dismissed.
```

Comment:
I purposely left this one out because we deprecated it. It's available in the code for backwards compatibility, but let's keep it out of the docs to avoid it being used going forward.